### DISCUSSION TITLE
Codespace option not showing after upgrading to teams

### DISCUSSION THREAD

**Discussion by jasperkennis** [community/community#25646] [Codespaces]
<p>Hi all,</p>
<p>I wanted to give codespaces a go so I created an org and set payed for a teams account. I attempted to follow the quick start found <a href="https://docs.github.com/en/codespaces/getting-started/quickstart" rel="noopener nofollow ugc">here</a>, but get stuck right after creating the project from the template because there is no codespace option available to me. I’m not sure what I’m doing wrong; I have a paid team account, what more does it need?</p>
<p>Thanks in advance, Jasper</p>

**Comment by sudo-nautilus** [comment]
<p>Try clicking the <code>Code</code> drop-down menu and <code>Codespaces</code> tab may get displayed.</p>

**Comment by jasperkennis** [comment]
<p>Yeah, I’m aware that it should show there, but it isn’t showing…</p>

**Comment by tiankuan93** [comment]
<p>I ran into the same problem.</p>

**Comment by miketikh** [comment, marked as answer]
<p>Found this here: <a href="https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/enabling-codespaces-for-your-organization" class="inline-onebox" rel="noopener nofollow ugc">Enabling Codespaces for your organization - GitHub Docs</a></p>
<p>Apparently you need to enable codespace access, but when you follow the instructions above and click on “codespaces,” it says “Your organization doesn’t have access, click here to request.”</p>
<p>I emailed them, so we’ll see, but would be nice if the guide specified that codespaces aren’t available for everyone, instead of just telling you to upgrade to Team.</p>
